u_conn_test.go

```diff
@@ -290,7 +290,7 @@ func testUTLSHandshakeClientECDHE_RSA_AES256_GCM_SHA256(t *testing.T, helloID Cl
 	opensslCipherName := "ECDHE-RSA-AES128-GCM-SHA256"
 	test := &clientTest{
 		name:    "UTLS-" + opensslCipherName + "-" + helloID.Str(),
-		args: []string{"openssl", "s_server", "-cipher", opensslCipherName},
+		args: []string{"-cipher", opensslCipherName},
```

Comment:
is there a reason to have `-cipher` here, but `-ciphersuites` below?

Reply:
If I'm correctly understanding the docs, `-cipher` is for tlsv1.2 and below and `-ciphersuites` is for tlsv1.3.